composer.json

```
@@ -71,7 +69,9 @@
     },
     "classmap": [
       "app/AppKernel.php",
-      "app/AppCache.php"
+      "app/AppCache.php",
+      "sources/Afup/Bootstrap/_Common.php",
+      "sources/Afup/Bootstrap/commonStart.php"
```

Tu es sûr de ça ? Il me semblait que pour charger des fichiers en procédural il fallait utiliser files et non classmap :

```suggestion
      "app/AppCache.php"
    ],
    "files": [
      "sources/Afup/Bootstrap/_Common.php",
      "sources/Afup/Bootstrap/commonStart.php"
```